plenum/common/messages/node_messages.py

```diff
@@ -282,10 +281,11 @@ class MessageReq(MessageBase):
     """
     Purpose: ask node for any message
     """
+    allowed_types = {LEDGER_STATUS, CONSISTENCY_PROOF, PREPREPARE,
```

Comment:
In TDD the tests is one of the main (and sometimes the only) peace of requirement. 
You declare in tests how the should behaves, what is the API, and what is the contract.
If someone changes the Contract (Supported message types), then the test will fail indicating that we changed API/requirement. This is additional check that the change is intentional.